### PR TITLE
Api helloworld

### PR DESCRIPTION
**Pull request by spectroalias**

### CODE REVIEW

File: demo/apiDrfDocker/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ApidrfdockerConfig(AppConfig):
```
Comment (tucosaurus) [inline demo/apiDrfDocker/apps.py:4]: Read about PEP8 guidelines in python

**Comment by tucosaurus** [comment]
Each change that goes in a PR should ideally be one unit. Only commit the basic Django setup in this PR.
Create a separate PR for dockerizing the project.

**Review by tucosaurus — CHANGES_REQUESTED** [review]
already commented on the required changes

File: demo/apiDrfDocker/apps.py (hunk 1 above)
Comment (spectroalias) [reply demo/apiDrfDocker/apps.py:4]: @tucosaurus , got it.

**Pull request closed without merging**